### DISCUSSION TITLE
Meeting cadence moving forward

### DISCUSSION THREAD

**Discussion by scottrigby** [open-gitops/project#18] [Ideas] [answered]
Next GitOps WG monthly meeting is [Wednesday Aug 18, 18:00 GMT](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.s24jr4bhecme). First item on the agenda is our WG meeting cadence moving forward.

## Participate

1. Please join us if you can!
2. If you can't join the meeting, or in either case, please reply here with your thoughts and needs for this topic
3. Follow this discussion after the meeting to participate in the decision about our WG meeting cadence moving forward

The plan is to collect members needs and opinions here in one place for those who both can and can't make this monthly meeting in person. Please weigh in here before the meeting if you know you can't make it in person so your views can be discussed. Views voiced during the meeting only will be summarized here as well, so we can make an async decision as a group [per our WG governance](https://github.com/gitops-working-group/gitops-working-group/blob/main/GOVERNANCE.md#decision-making), to include as many people as possible.

## Why change the meeting cadence?
The group has spoken, while the per-committee meetings have been fruitful, major goals have been won and these separate meetings are seeing diminishing returns. In the meantime we’ve formed a very productive working group. The idea to consolidate our 3 separate WGW meetings into one recurring meeting was discussed in every recent meeting over the past month and seemed have broad consensus.

## What should the cadence be?
- Several people have proposed we have one fortnightly meeting with everyone
- Others have proposed one early and one late meeting, alternating weekly, to accommodate east and west timezones
- Other ideas?

The goal is to accommodate as many existing members as possible, while hopefully opening up to new people if anything, in a way that allows us to continue making collaborative progress on the WG goals.

**Comment by jlbutler** [comment]
I'm +1 on a bi-weekly (twice monthly), with alternating times to accommodate folks on either side of the timezone spectrum.

**Reply by roberthstrand** [reply]
Just to interject, I'm in a pretty different timezone than both east and west coast of the US, and the time we have now works perfectly fine for me. Unless we have a lot of people that are missing out because they are in a timezone even further away than CEST, I'd say have two meetings but I don't think we need that at the moment.

**Comment by scottrigby** [comment, marked as answer]
📣 **Update:**
- 🗓 The WG now meets every other Wednesday at [18:00 GMT](https://greenwichmeantime.com/time-gadgets/time-zone-converter/)
- 👩‍💻 Join using [this Zoom link](https://zoom.us/j/93779536510?pwd=TEFzbGRzREI3MVlkZmluemVkMEhHdz09)
- ✏️ See [Agenda and Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) for next and and all previous meetings
